src/theme/components/checkbox.ts

```diff
@@ -0,0 +1,79 @@
+import { checkboxAnatomy } from "@chakra-ui/anatomy";
+import { createMultiStyleConfigHelpers, defineStyle } from "@chakra-ui/react";
+
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(checkboxAnatomy.keys);
+
+const baseStyleControl = defineStyle(() => {
+  return {
+    borderRadius: "2px", // change the border radius of the control
+    border: "2px solid",
+    borderColor: "gray.700",
+    flexShrink: 0,
+    backgroundColor: "white",
+    color: "white",
+    gap: "10px",
+    _checked: {
+      padding: "2px",
```

Comment:
Removing the padding here and in the other pseudo styles didn't seem to have any effect. See if deleting it breaks anything. Best to do the minimum styling to get the required behavior.